content/en/storage-providers/curio/config.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Config"
+description: "This guide provides detailed information about Curio configuration"
+date: 2024-03-22T19:32:32+04:00
+lastmod: 2024-03-22T19:32:32+04:00
+draft: false
+menu:
+  storage-providers:
+    parent: "curio"
+weight: 135
+---
+
+The configuration for Curio is stored in the HarmonyDB in a table called `harmony_config`.
+When a Curio node is started, one or more layer names are supplied to get the desired configuration for the node.
+
+## Configuration Layers
+Configuration layers provide a set of configuration parameters that determine how a system will operate.
+These layers can be defined at different levels, meaning that higher layer can override the lower layer, and the system will behave based on the final stacked output.
+
+Configuration layers can be arranged in a hierarchy, often from `base`(most general) to most specific.
+The `base` layer defines default configuration values.
+More specific layers override these defaults with more targeted configurations.
+
+For example, in a simple three-layer configuration system, layers could be organized in the following order:
+Base Layer - This is the most general layer. It normally provides default settings.
+Miner Layer - This layer overrides the miner ID from emtpy to a `M1`.
+Task Layer - This layer enables SDR tasks.
+
+If a Curio node is started with above 3 layers then it will perform SDR tasks for Miner ID `M1` and will use default values of any other configuration parameter.
```

Comment:
If you define miner IDs in layers instead of `base`, then only the machines running those layers will operate on those miner IDs.